testing/test_collection.py

```diff
@@ -1860,6 +1860,27 @@ def test_pyargs_collection_tree(pytester: Pytester, monkeypatch: MonkeyPatch) ->
     )
 
 
+def _can_create_symlink(base: Path) -> bool:
```

Comment:
Instead of this, I think a better fix would be to replace the `assert symlink_path.is_symlink()` with a skip if it's not a symlink. That's a direct check.